tests/test_reduce_baselines.py

```
@@ -0,0 +1,132 @@
+    )
+
+    output = reduce_baselines(raw, out, slack=1.5, source="ubuntu-latest-ci")
+    data = json.loads(out.read_text(encoding="utf-8"))
+    groups = data["groups"]
+
+    assert groups["parse"]["test_list_workspace_projects_nocache[composers-50]"] == pytest.approx(0.075)
+    assert groups["export"]["test_post_export_zip[composers-10]"] == pytest.approx(0.015)
+    assert groups["search"]["test_search_full_corpus"] == pytest.approx(0.06)
+    assert data["machine"] == "Linux"
+    assert "ubuntu-latest CI benchmark-results.json" in data["_note"]
+    assert "1.5× slack" in data["_note"]
```

Ruff (0.15.18)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)